src/execution_plan/ops/shared/scan_functions.h

```diff
@@ -8,16 +8,18 @@
 
 // Storage struct for label data in node and index scans.
 typedef struct {
-	const char *alias;   // Alias of the node being traversed.
-	const char *label;   // Label of the node being traversed.
-	int label_id;        // Label ID of the node being traversed.
+	const char *alias;  // alias of the node being traversed
+	const char *label;  // label of the node being traversed
+	int label_id;       // label ID of the node being traversed
+	int *candidates;   // candidates for swapping in Label-Scan optimization
```

Comment:
`int` should be replaced with `LabelID` for both `candidates` and `label_id`
Also make sure `//` are all aligned.

Lastly should this become: `const int *candidates` ?